java/sage/media/format/SubpictureFormat.java

```diff
@@ -76,6 +86,18 @@ public String getFullPropertyString()
       sb.append(ancillaryPageId);
       sb.append(';');
     }
+    if(this.forced == true)
```

Comment:
Just change this whole block to be:

sb.append("forced=");
sb.append(forced);
sb.append(';');

Reply:
You missed this comment....